Jenkinsfile.mechanical

```diff
@@ -0,0 +1,33 @@
+def utils, streams
+node {
+    checkout scm
+    utils = load("utils.groovy")
+    streams = load("streams.groovy")
+}
+
+properties([
+    pipelineTriggers(streams.get_push_trigger() + [
+        // also run every 30m: we could increase the interval more if we start
+        // triggering on the relevant fedmsgs.
+        cron("H/30 * * * *")
+    ])
+])
+
+node {
+    change = checkout(
+        [$class: 'GitSCM',
+         userRemoteConfigs: [
+            [url: 'https://github.com/coreos/fedora-coreos-config']
+         ],
+         branches: streams.as_branches(streams.prod)
```

Comment:
streams.production

Reply:
Heh, yeah caught those while testing this morning.